images/openvswitch/Dockerfile

```diff
@@ -4,18 +4,13 @@
 # The standard name for this image is openshift/openvswitch
 #
 
-FROM centos:centos7
+FROM openshift/origin-base
```

Comment:
This particular image has no ties to OpenShift and I believe @smarterclayton requested that it act as a base image rather than depending on origin-base when I originally submitted it. I'm indifferent on the matter.